src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -3385,8 +3415,14 @@ void Fishing_UpdateFish(Actor* thisx, PlayState* play2) {
                 chance *= 5.0f;
             }
 
+#if OOT_DEBUG
             if (((this->timerArray[0] == 1) || (Rand_ZeroOne() < chance)) &&
-                ((Rand_ZeroOne() < (this->perception * multiplier)) || ((this->isLoach + 1) == KREG(69)))) {
+                ((Rand_ZeroOne() < (this->perception * multiplier)) || ((this->isLoach + 1) == KREG(69))))
+#else
+            if (((this->timerArray[0] == 1) || (Rand_ZeroOne() < chance)) &&
+                (Rand_ZeroOne() < (this->perception * multiplier)))
+#endif
+            {
```

Comment:
I forgot what we made of this last time / in which PR it was
```suggestion
            if (((this->timerArray[0] == 1) || (Rand_ZeroOne() < chance)) &&
                ((Rand_ZeroOne() < (this->perception * multiplier))
#if OOT_DEBUG
                 || ((this->isLoach + 1) == KREG(69))
#endif
                     )) {
```

Reply:
id say just do whatever you think is best for now and we will decide on something and consistently use it after retail is matched